shell/pages/c/_cluster/manager/drivers/kontainerDriver/_id.vue

```diff
@@ -0,0 +1,12 @@
+<script>
+import ResourceDetail from '@shell/components/ResourceDetail';
+
+export default {
+  name:       'KontainerDriverEditPage',
+  components: { ResourceDetail },
+};
+</script>
+
+<template>
+  <ResourceDetail />
```

Comment:
When I go to edit an existing kontainer or node driver, the form never loads and I get some errors in the console: 
![Screenshot 2024-03-21 at 12 37 51 PM](https://github.com/rancher/dashboard/assets/42977925/62188889-5936-439d-8f5e-75dcf6c0b753)

From what I can see, you need `resource-override` set in kontainerDriver/_id.vue and nodeDriver/_id.vue as you have with the create pages.

Reply:
That's so weird. Because I just removed it because it doesn't make a difference when I run it. I don't see the error you are getting.

Reply:
I'm not sure what's up with that. I was able to see it consistently with both node and kontainer drivers and verified that my proposed solution worked